### ISSUE TITLE
Restore Fails

### ISSUE THREAD

**Issue by purplepoet01** [i7/counterfeit-monkey#191]
Every time I try to restore a saved file, the site tells me restore failed. Not sure why.

**Comment by angstsmurf** [comment]
Thanks for reporting this! Which site is it you are playing on?

**Comment by whatwhatwhatwhatwoahhh** [comment]
I have the same problem. Im using the link from ifdb
[https://iplayif.com/?story=https%3A%2F%2Fgithub.com%2Fi7%2Fcounterfeit-monkey%2Freleases%2Fdownload%2Fr10%2FCounterfeitMonkey-10.gblorb](url)

**Comment by angstsmurf** [comment]
Thanks!

I can't reproduce this, but I had problems in the past running out of disk space quota when running this in a browser.

What browser and OS is this? Did it happen the first time you tried to restore or did it use to work? Do you have a lot of saved positions?

**Comment by curiousdannii** [comment]
I wrote more at https://intfiction.org/t/counterfeit-monkey-restoring-fails-in-parchment/57013/2

TLDR: I can reproduce it, and there are improvements we can make to Parchment, but there's probably not really much to do on the CM side.

**Comment by curiousdannii** [comment]
I have made it so that Parchment now encodes files in base32768 for localStorage, so you should be able to store about 7 times more data there now. This is live on [iplayif.com](http://iplayif.com/) and from what I can see everything is working properly. Let me know if you see any bugs.

I haven’t made it delete old autosaves, but I intend to look at that soon.

**Comment by curiousdannii** [comment]
@angstsmurf You could look at bringing in this code into CM, it should work in 6M62: https://github.com/ganelson/inform/pull/64

**Comment by angstsmurf** [comment]
I incorporated your fix. It would be nice if somebody could confirm that it is working.
